image/pif.ksy

```diff
@@ -0,0 +1,58 @@
+meta:
+  id: pif
+  title: Portable Image Format
+  file-extension: pif
+  license: LGPL-2.1
+  ks-version: 0.10
```

Comment:
@armijnhemel It should work in 0.9 too, doesn't it?

Reply:
Yes, it should be. I just default to using the latest version. I will change this to 0.9.